sqlite-db/src/rollover/insert.rs

```diff
@@ -80,7 +80,7 @@ async fn insert_rollover_completed_event_data(
     dlc: &Dlc,
     funding_fee: FundingFee,
     complete_fee: Option<CompleteFee>,
-    offer_id: models::OrderId,
+    order_id: models::OrderId,
```

Comment:
🤔should this be order_id or offer_id? how can we tell which one of the two we need here?

Reply:
This one was super confusing for myself, because when this was introduced we treated the offer as order (i.e. we always spoke of the offer as order and wanted to rename it to offer), hence @bonomat opted for naming this `offer_id`. However, with the introduction of clear boundaries for offer and order as part of my commit this actually is related to the order, and this *is* an `OrderId` here. From a lifecylce point of view when this is created the order got filled.
Currently, the `offer_id` in the `Cfd` is just to identify the offer related to the `Cfd`, but not the `Cfd` itself. The `OrderId` identifies the `Cfd`, because if an order gets filled the taker creates the `Cfd` with a unique `OrderId`.

Note: The lifecycle of the `Cfd` can still be improved, we could introduce a dedicated `Order` and only once said `Order` is filled we create the actual `Cfd`. But I felt we can do that in another iteration on top of this work if we want to. So for now I kept the `cfds` table as a flattened out data conglomerate that includes all the information and the lifecylce start with creating the `Order` on the taker side. This means that, as before, we can have a `Cfd` in the database that was never actually filled by the maker - which is a bit weird.

Reply:
I created https://github.com/itchysats/itchysats/issues/2555 to shed some more light on this and outline how I envision to clean this up.